src/pgduckdb_duckdb.cpp

```diff
@@ -222,7 +222,7 @@ DuckDBManager::CreateConnection() {
 
 	auto http_file_cache_set_dir_query =
 	    duckdb::StringUtil::Format("SET http_file_cache_dir TO '%s';", CreateOrGetDirectoryPath("duckdb_cache"));
-	context.Query(http_file_cache_set_dir_query, false);
+	TryDuckDBQuery(context, http_file_cache_set_dir_query);
```

Comment:
I feel like this should probably just fail if it doesn't work.

Reply:
Agreed - since it wasn't the original behavior, @mkaruza wdyt?

Reply:
Yes, we build this extension statically with duckdb so it *should* be there, like @JelteF  said than something very strange is going if this fails.